src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? (`${partnerName} is my ${partner}`)
```

Comment:
Doesn't seem to be necessary to use additional parentheses here
```suggestion
          ? `${partnerName} is my ${partner}`
```

Reply:
I believe Serhii is right here